src/revChatGPT/ChatGPT.py

```diff
@@ -3,6 +3,9 @@
 import json
 import tls_client_for_chatGPT as tls_client
 import undetected_chromedriver as uc
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
```

Comment:
Is it possible to do this without selenium? Extra dependencies just for Microsoft login (which is quite rare) isn't worth it.

Reply:
Probably can’t be done without extra dependencies. But I use Microsoft Login on all my libraries cause no 2FA and no captchas needed so less cost for me, your call though.

Reply:
The same issue will be presented when wanting to use email/password login.

Reply:
Selenium is already installed with undetected_chromedriver.

Reply:
![image](https://user-images.githubusercontent.com/82155626/209726697-397336ba-19c1-4049-8cb0-9fa6732701d5.png)

Reply:
In this case, this is perfectly fine. Thank you

Reply:
No worries man, if you need anything or have any questions just hit me up on Discord.